snuba/datasets/processors/errors_processor.py

```diff
@@ -311,6 +312,8 @@ def extract_stacktraces(
         frame_stack_levels = []
         exception_main_thread = None
 
+        check_exception_main_thread = get_config("check_exception_main_thread", True)
```

Comment:
Configs should either be numbers or strings. So use 0/1 instead for this.

Reply:
Done but Booleans are used multiple types in the code base, for example
`if get_config("skip_seen_offsets", False):`
`state.get_config("enable_events_readonly_table", False)`
Etc...